cmd/eksctl-anywhere/cmd/options.go

```diff
@@ -2,16 +2,22 @@ package cmd
 
 import (
 	"fmt"
+	"os"
 	"path/filepath"
+	"strings"
 
+	"github.com/aws/eks-anywhere/pkg/api/v1alpha1"
 	"github.com/aws/eks-anywhere/pkg/cluster"
+	"github.com/aws/eks-anywhere/pkg/config"
+	"github.com/aws/eks-anywhere/pkg/providers/cloudstack/decoder"
 	"github.com/aws/eks-anywhere/pkg/version"
 )
 
 type clusterOptions struct {
 	fileName             string
 	bundlesOverride      string
 	managementKubeconfig string
+	installPackages      string
```

Comment:
One of my concerns is whether including the installPackages in clusterOptions makes sense or if this should be wrapped up in a separate structure. 

Having it in clusterOptions helps ensure directoriesToMount runs which is used by create, update, and delete cmds.

Reply:
I think this makes total sense. Especially since it can be passed in as a flag to our commands, similar to every other field in this struct